addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -858,6 +858,19 @@ export class OdooEditor extends EventTarget {
         this._computeHistoryCursor();
         return this._applyCommand(...args);
     }
+    bindExecCommand(element) {
```

Comment:
The method name being a little cryptic, I went to the method definition hoping to find a docstring.
```
Find all descendants of `element` with a `data-call` attribute and bind them on mousedown to the execution of the command matching that attribute.
```
It's a tongue twister but it's something...

Reply:
done